pkg/crc/machine/machine.go

```diff
@@ -594,11 +594,11 @@ func Status(statusConfig ClusterStatusConfig) (ClusterStatusResult, error) {
 		ocConfig := oc.UseOCWithConfig(statusConfig.Name)
 		operatorsStatus, err := oc.GetClusterOperatorStatus(ocConfig)
 		if err != nil {
-			result.Success = false
-			result.Error = err.Error()
-			return *result, errors.New(err.Error())
+			logging.Warn(err.Error())
```

Comment:
See the comment I made here:
https://github.com/code-ready/crc/pull/1195#issuecomment-620502328
>The warning which is shown is not a user-readable error, so I don't think it makes sense to show as WARN. If it was saying that it seems there was an issue with VM creation, then that's something the user can investigate, and it's a better WARN message.

Reply:
@cfergeau do you think then a debug make more sense than have it as part of warn ?